packages/app-builder-lib/templates/linux/after-install.tpl

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 # Link to the binary
-ln -sf '/opt/${sanitizedProductName}/${executable}' '/usr/bin/${executable}'
+ln -sf $'/opt/${sanitizedProductNameInBashScript}/${executable}' '/usr/bin/${executable}'
```

Comment:
What's the additional `$` operating as here?
What if we changed the command to use `"` instead of `'` so that apostrophes are retained?
```
ln -sf "/opt/${sanitizedProductName}/${executable}" "/usr/bin/${executable}"
chmod 4755 "/opt/${sanitizedProductName}/chrome-sandbox" || true
```

Reply:
You can refer to [this link](https://wiki.bash-hackers.org/syntax/quoting#ansi_c_like_strings) to learn more about ```$'string'```. It can allow escape sequences to be decoded.

Using ```"``` instead of ```'``` can also solve this problem. Do you want me to push another commit that uses this way?

Reply:
Let's use the `"` route for simplicity, that way we keep the template to a minimal number of vars.